src/main/scala/com.snowplowanalytics.dataflow/streaming/SimpleEvent.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.dataflow.streaming
+
+// Java
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.DateTime
+
+// json4s
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+import org.json4s.ext.JodaTimeSerializers.all
+
+// This project
+import storage.BucketingStrategy
+
+//Scala
+import scala.util.Try
+
+/**
+ * Companion object for creating a SimpleEvent
+ * from incoming JSON
+ */
+object SimpleEvent {
+
+  //Singleton object to be able to parse our DateTime format
+  case object CustomDateTimeSerializer extends CustomSerializer[DateTime](format => (
+    {
+      case JString(s) => new DateTime(s)
+      case JNull => null
+    },
+    {
+      case d: DateTime => JString(format.dateFormat.format(d.toDate))
+    }
+  ))
```

Comment:
I was getting an error using the json4s-ext joda time serializer, because (from what I understood) it expects a specific date format. I googled for someone with the same problem and found this solution, which in fact renders json4s-ext unnecessary.

Is there a better way to do it? Like allowing different formats in the json4s-ext joda time serializer?